.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 * @spicetify/marketplace
```

Comment:
I'm not sure what this line does tbh. Does it need to be adjusted as well or no?

Reply:
Nah, that just means every file in the repository is owned by the `@spicetify/marketplace` team